util/progress/multiwriter.go

```diff
@@ -100,3 +109,24 @@ func (ps *MultiWriter) writeRawProgress(p *Progress) error {
 func (ps *MultiWriter) Close() error {
 	return nil
 }
+
+func (ps *MultiWriter) contains(pw rawProgressWriter) bool {
+	ps.mu.Lock()
+	defer ps.mu.Unlock()
+	_, ok := ps.writers[pw]
+	if ok {
+		return true
+	}
+
+	for w := range ps.writers {
+		w, ok := w.(*MultiWriter)
+		if !ok {
+			continue
+		}
+		if w.contains(pw) {
+			return true
+		}
```

Comment:
nit:
```suggestion
		if w, ok := w.(*MultiWriter); ok && w.contains(pw) {
			return true
		}
```

Reply:
that's an upstream nit :zany_face: 

I would rather not introduce additional changes into our fork.